bitnami/jupyterhub/templates/hub/deployment.yaml

```
@@ -0,0 +1,227 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: hub
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  name: {{ template "common.names.fullname" . }}-hub
```

Add the namespace (this applies to every manifest):
```suggestion
  name: {{ template "common.names.fullname" . }}-hub
  namespace: {{ .Release.Namespace | quote }}
```

---

Done